x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_details_page/components/agent_details/agent_details_integrations.tsx

```diff
@@ -153,17 +193,29 @@ export const AgentDetailsIntegration: React.FunctionComponent<{
                   {packagePolicy.name}
                 </EuiLink>
               </EuiFlexItem>
+              {showNeedsAttentionBadge && (
+                <EuiFlexItem grow={false}>
+                  <EuiBadge color="#BD271E" iconType="alert" iconSide="left">
```

Comment:
`color="danger"`

Reply:
Not exactly, danger color for EuiBadge is not the red one (saw that on docs).

Reply:
Oh yes! You're right! That's odd for the same token name to have different colors. Perhaps let the EUI team know? 🙏

Reply:
I'd suggest that we the `euiTheme` here to pull the color out. Something like `color={euiTheme.colors.danger}`. You can import and use `useEuiTheme` from `@elastic/eui`